### PR TITLE
clairctl: Add delete command

### PR DESCRIPTION
**Pull request by crozzy**
Delete capability was added to the API recently, this adds the ability to invoke that endpoint directly through clairctl.

Signed-off-by: crozzy <joseph.crosland@gmail.com>

### CODE REVIEW

File: cmd/clairctl/delete.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"os"
+
+	"github.com/quay/clair/v4/internal/httputil"
+	"github.com/quay/claircore"
+	"github.com/urfave/cli/v2"
```
Comment (hdonnay) [inline cmd/clairctl/delete.go]:
```suggestion
	"github.com/quay/claircore"
	"github.com/urfave/cli/v2"

	"github.com/quay/clair/v4/internal/httputil"
```

File: cmd/clairctl/delete.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+)
+
+var DeleteCmd = &cli.Command{
+	Name:        "delete",
+	Description: "Delete index reports for given manifest digests",
+	Action:      deleteAction,
+	Usage:       "deletes index reports for given manifest digests",
+	ArgsUsage:   "manifest digests...",
```
Comment (hdonnay) [inline cmd/clairctl/delete.go]:
```suggestion
	ArgsUsage:   "digest...",
```

File: cmd/clairctl/delete.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	"os"
+
+	"github.com/quay/clair/v4/internal/httputil"
+	"github.com/quay/claircore"
+	"github.com/urfave/cli/v2"
+)
+
+var DeleteCmd = &cli.Command{
+	Name:        "delete",
+	Description: "Delete index reports for given manifest digests",
```
Comment (hdonnay) [inline cmd/clairctl/delete.go]:
```suggestion
	Description: "Delete index reports for given manifest digests.",
```

**Review by hdonnay — CHANGES_REQUESTED** [review]
Some nits

File: cmd/clairctl/delete.go (hunk 1 above)
Comment (crozzy) [reply cmd/clairctl/delete.go]: Huh, I guess goimports gets confused when the cmd is different

**Review by hdonnay — APPROVED** [review]

**Pull request merged**